src/plugins/guided_onboarding/server/saved_objects/guided_setup.ts

```diff
@@ -22,12 +18,26 @@ export const guidedSetupSavedObjects: SavedObjectsType = {
   mappings: {
```

Comment:
As already mentioned, we might not need all the mappings. Here is a comment from the POC PR https://github.com/elastic/kibana/pull/138611#discussion_r971731871. We can address this as a follow up though